manifests/core-bases/base/federation-configmap.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: federation-config
+data:
+  module-federation-config.json: |
+    [
+      {
+        "name": "modelRegistry",
+        "remoteEntry": "/remoteEntry.js",
+        "authorize": true,
+        "tls": false,
```

Comment:
You should enable tls with a Service Serving Certificate

Reply:
Done, thanks a lot for the feedback didn't thought of that at first.